releases/Process/Major Release Process.md

```diff
@@ -59,91 +56,108 @@ The release process is managed by sig-release. To request an update to this docu
+* All bugs in the Milestone must be closed before the release can go out. 
 
+*Exit Criteria:*
+*1. All of the [General Steps](#1.1-General-Steps) are complete.*
+*2. Ideally, but not required. All of the planning tasks are complete on the project board. If not all tasks are complete, it is up to the release manager and co-release manager to understand and accept any risk before moving onto Phase 2.*
 
-## Phase (2 of 4): Pre Release (includes Code Stabilization)
-*Intent: Accomplish all tasks needed to prepare the codebase for release. Complete all documentation and marketing materials.* 
 
-Begins 6-8 weeks prior to release. This Phase may begin before previous phases are completed, at the discretion of the Release Manager.
-Entry Criteria: Project board exists and Pre-Release tasks were created during the Release Planning Phase. 
+## Phase (2 of 4): Pre Release (includes Code Stabilization)
+*Intent: Accomplish all tasks needed to prepare the codebase for release. Complete all documentation and marketing materials.
+Begins: 6-8 weeks prior to release. This Phase may begin before previous phases are completed, at the discretion of the Release Manager.
+Entry Criteria: Project board exists and Pre-Release tasks were created during the Release Planning Phase.*
 
 ### 2.1 Communication during Stabilization
 It is crucial to maintain strong communication with the community and throughout stabilization. This includes but not limited to:
- - **Release Manager** At least 1 week before stabilization, posting a notification to sig-all and sig-release giving them a heads up.  For example: https://discord.com/channels/805939474655346758/816043864858951710/1011404890364444712
- - **Release Manager** When the stabilization branch is created, post a notification to sig-all and sig-release. This is already listed as a step in the stabilization process
+ - **Release Manager** At least 1 week before stabilization, posting a notification to #sig-all and #sig-release Discord channels giving them a heads up.  For example: https://discord.com/channels/805939474655346758/816043864858951710/1011404890364444712
+ - **Release Manager** When the stabilization branch is created, post a notification to #sig-all and #sig-release Discord channels. This is listed as a step in the stabilization process, but listing it again here 
  - **Release Manager** Midway through the stabilization, posting a notification informing the community of the remaining time.
- -  **Release Manager** work with QA/sig-testing to ensure we have regular updates in Discord as to the number of outstanding bugs and how we are doing in terms of stabilizing the release.
+ -  **Release Manager** communicate regularly with each SIG in Discord to ensure they are aware of the outstanding bugs in the Stabilization branch (the bugs can be found in the Github milestone). 
```

Comment:
What's the regular cadence that works in our previous release? Maybe it's worth sharing so the RM knows what works before @tonybalandiuk

Reply:
It currently takes about 5 weeks to stabilize the branch, test it, and declare it "Stable". I can include that.

Reply:
included it in my Jan 3 commit

Reply:
Oh, what I refer to is the communication regular cadence, not the entire project. "Communicate regularly" is vague in my opinion so I propose to provide an example that we see work in the previous release. Thoughts? @tonybalandiuk